src/app/helptext/system/ssh-connections.ts

```diff
@@ -2,37 +2,64 @@ import { T } from '../../translate-marker';
 
 export default {
     name_placeholder: T('Name'),
-    name_tooltip: T(''),
+    name_tooltip: T('Descriptive name of this SSH connection. SSH \
+ connection names must be unique.'),
 
     setup_method_placeholder: T('Setup Method'),
-    setup_method_tooltip: T(''),
+    setup_method_tooltip: T('How to configure the connection\:<br><br> \
+ <i>Manual</i> requires configuring authentication on the remote system. \
+ This can require copying SSH keys and modifying the <i>root</i> user \
+ account on that system.<br><br> \
+ <i>Semi-automatic</i> is only functional when configuring an SSH \
+ connection between FreeNAS systems. After authenticating the \
```

Comment:
But it's not just FreeNAS, also TrueNAS.
```suggestion
 connections between FreeNAS or TrueNAS systems. After authenticating the \
```